.golangci.yml

```diff
@@ -9,13 +9,10 @@ linters:
     "gofmt",
     "gosec",
     "staticcheck",
-    "deadcode",
     "dupl",
     "errcheck",
-    "structcheck",
     "unparam",
     "unused",
-    "varcheck",
```

Comment:
Why drop these?

Reply:
It shows warnings:
```
❯ make lint
WARN [runner] The linter 'deadcode' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter.  Replaced by unused. 
WARN [runner] The linter 'varcheck' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter.  Replaced by unused. 
WARN [runner] The linter 'structcheck' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter.  Replaced by unused. 
WARN [linters_context] structcheck is disabled because of generics. You can track the evolution of the generics support by following the https://github.com/golangci/golangci-lint/issues/2649. 
```